students/geonwoo/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+class Sign_up(models.Model):
+    name        = models.CharField(max_length=10)
+    email       = models.CharField(max_length=30)
+    password    = models.CharField(max_length=20)
+    phoneNumber = models.CharField(max_length=20)
```

Comment:
```suggestion
    phone_number = models.CharField(max_length=50)
```
python에서 변수명은 카멜 케이스가 아닌 스네이크 케이스를 사용합니다.

- phoneNumber -> phone_number
- max_length가 조금 작은 것 같습니다. 50으로 늘려주세요.